src/MemoryForm/MemoryForm.js

```diff
@@ -8,76 +8,115 @@ import {
   Modal,
   Button,
   TouchableOpacity,
-  TextInput
+  TextInput,
+  Alert,
 } from "react-native";
-import { editMemory } from "../apiCalls";
+import { editMemory, deleteMemory } from "../apiCalls";
 // import { TextInput } from 'react-native-paper';
-import { Colors } from "react-native/Libraries/NewAppScreen";
-
 
 // Edit is clicked from within a memory
 // Flip a bool in Memory when edit mode is activated and
 // form will render instead of static memory
 
-
-export default MemoryForm = ({memory, setSelectedMemory, setModalVisible, modalVisible, setEditMode, editMode}) => {
-  const [updatedDescriptionText, setUpdatedDescriptionText] = useState(memory.description);
+export default MemoryForm = ({
+  memory,
+  setSelectedMemory,
+  setModalVisible,
+  modalVisible,
+  setEditMode,
+  editMode,
+  memoryUpdatedBool,
+  setMemoryUpdatedBool,
+}) => {
+  const [updatedDescriptionText, setUpdatedDescriptionText] = useState(
+    memory.description
+  );
   const [updatedAromaText, setUpdatedAromaText] = useState(memory.aromas);
   const [updatedMemory, setUpdatedMemory] = useState({});
-  // const [formVisible, setFormVisible] = useState(true);
-  
+
   useEffect(() => {
-    // console.log('updatedMemory :>> ', updatedMemory);
-    console.log('updatedDescriptionText :>> ', updatedDescriptionText);
-    console.log('updatedAromaText :>> ', updatedAromaText);
     if (Object.keys(updatedMemory).length) {
-      console.log('if there is a new memory', updatedMemory);
-      editMemory(memory.id, updatedMemory)
-      .then(response => console.log(response))
+      editMemory(memory.id, updatedMemory);
       setSelectedMemory(null);
       setModalVisible(!modalVisible);
     }
-  }, [updatedMemory])
+  }, [updatedMemory, modalVisible]);
+
+  const displayDeleteConfirmation = () => {
+    Alert.alert(
+      "Delete This Memory?",
+      "Are you sure you want to delete this memory?",
+      [
+        {
+          text: "NO",
+          onPress: () => console.warn("NO Pressed"),
+          style: "cancel",
+        },
+        {
+          text: "YES",
+          onPress: () => {
+            deleteMemory(memory.id)
```

Comment:
Good functionality here. We need to figure out the rendering but so far this is great.